src/include/optimizer/statistics/column_stats.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include <sstream>
+
+#include "catalog/catalog_defs.h"
+#include "common/macros.h"
+
+namespace terrier::optimizer {
+
+class ColumnStats {
+ public:
+  ColumnStats(catalog::db_oid_t database_id, catalog::table_oid_t table_id, catalog::col_oid_t column_id,
+              const std::string &column_name, bool has_index, size_t num_rows,
```

Comment:
Why do we need the column name if we have the `col_oid_t`?

Reply:
In TableStats, there is a set of functions that take in the column name rather than col_oid_t (the justification is to support intermediate tables).